packer/scripts/versions_to_json.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+require 'json'
+
+def print_ansible_variable_json(version_hash)
+  foreman_branch = version_hash['foreman'] == 'nightly' ? 'develop' : "#{version_hash['foreman']}-stable"
+  katello_branch = version_hash['katello'] == 'nightly' ? 'master' : "KATELLO-#{version_hash['katello']}"
+  ansible_options = {
+    'katello_repositories_version': version_hash['katello'],
+    'foreman_repositories_version': version_hash['foreman'],
+    'pulpcore_repositories_version': version_hash['pulpcore'],
+    'pulp_repositories_version': version_hash['pulp'],
+    'foreman_installer_options': ["-katello-devel-katello-scm-revision=#{katello_branch}", "--katello-devel-foreman-scm-revision=#{foreman_branch}"]
```

Comment:
```suggestion
    'foreman_installer_options': ["--katello-devel-katello-scm-revision=#{katello_branch}", "--katello-devel-foreman-scm-revision=#{foreman_branch}"]
```

Reply:
still in the middle of my first full test  ;)   surprisingly, this did not error out.... i'm curious why.  Will do some digging

Reply:
apparently ansible doesn't actually parse json in single quotes, it has to be in double quotes or in a file.  Playing around with a solution